content/_posts/2020-04-06-template-oriented-programming.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Template-Oriented-Programming to Ship Faster
+header-image: https://www.dropbox.com/s/249l72ehauel1np/guanlet-2.gif
```

Comment:
The previous suggested image can be used here as well.

Reply:
Thanks, Done!